### ISSUE TITLE
Playbook 2.0

### ISSUE THREAD

**Issue by overath2** [noQ-sweden/noq#668]
**Story**
As a noQer i need to understand what we are talking about internally so i can contribute. 

**Acceptance Criteria (checklist)**
- [ ]  Playbook in English 
- [ ]  Language section in Playbook updated with output from this story.
- [ ]  Plug-in to Slack for translation added.

**Other info:** 
Ex: Links, Solutions, best practices etc.

- [ ]  **Definition of Ready OK?**
Also called DoR, our generic process for checking if this is ready for start. See our Playbook here for details: https://docs.google.com/document/d/1ou6wr1sI9IJZaGGSMn_SJvFru5FJQKMDRZY16kBmx5Y/edit#heading=h.3rdcrjn

- [ ]  **Definition of Done OK?**
Also called DoD, our generic process for checking this is done in the right way. See our Playbook here for details: https://docs.google.com/document/d/1ou6wr1sI9IJZaGGSMn_SJvFru5FJQKMDRZY16kBmx5Y/edit#heading=h.23ckvvd

**Comment by overath2** [comment]
Can you help @DMould123 ?

**Comment by DMould123** [comment]
Yes, I am happy to help with this

**Comment by overath2** [comment]
Excellent. Lets do in next sprint.

**Comment by overath2** [comment]
Can this be don prior 15/12 @DMould123 ?

**Comment by DMould123** [comment]
Yes. I have translated the parts; 
Psykologisk trygghet 47 

Medlemskap 47 

Beslut 48 

Hur styrelsen fattar beslut 49 

Hur alla andra fattar beslut 49 
Länkar till partners och information mm 50  that you mentioned in the email and I am working my way throught the rest of the document. I will be able to finish it by 15/12     On Wednesday 13 November 2024 at 15:07:39 CET, Ove Holmberg (noQ) ***@***.***> wrote:  
 
 


Can this be don prior 15/12 @DMould123 ?

**Comment by overath2** [comment]
Excellent!

**Comment by overath2** [comment]
Can you put this ticket in progress? I will be wingmate.

**Comment by DMould123** [comment]
I am finished now with the document, I would suggest changing Missing people to Inactive members?

**Comment by overath2** [comment]
Yes this is a good idea. Although member = paying person for membership.
What about abscent? Inactive is perhaps the status for people on pause?
We need english words for the diffrent statuses:

- Intressent (som har anmält sig till infomöte)
- Jobbsökande (som har varit på infomöte och ansökt)
- Onboardare (som går onboarding)
- Volontär (som jobbar med noQ)
- Stödperson (som jobbar med noQ på stan men inte har noQ mail)
- Paus (person på paus)
- Missing (person som inte svarar på mail mm)
- Medlem (person som betalar medlemsavgiften till noQ)

**Comment by DMould123** [comment]
- Interested Party (who has signed up for an information meeting)
   - Jobseeker (who has attended an information meeting and applied)
   - Onboardee (who is going through onboarding)
   - Volunteer (who works with noQ)
   - Support Person (who works with noQ in the field but does not have a noQ email)
   - Inactive (person on a break)
   - Absent (person who does not respond to emails, etc.)
   - Member (person who pays the membership fee to noQ)   


    On Friday 13 December 2024 at 23:48:52 CET, Ove Holmberg (noQ) ***@***.***> wrote:  
 
 


Yes this is a good idea. Although member = paying person for membership.
What about abscent? Inactive is perhaps the status for people on pause?
We need english words for the diffrent statuses:
   
   - Intressent (som har anmält sig till infomöte)
   - Jobbsökande (som har varit på infomöte och ansökt)
   - Onboardare (som går onboarding)
   - Volontär (som jobbar med noQ)
   - Stödperson (som jobbar med noQ på stan men inte har noQ mail)
   - Paus (person på paus)
   - Missing (person som inte svarar på mail mm)
   - Medlem (person som betalar medlemsavgiften till noQ)

**Comment by overath2** [comment]
Go! Please implement then ping for review.

**Comment by patrik-pwned** [comment]
@overath2 in review just nu :)

**Comment by overath2** [comment]
Great work!
